OpenRA.Game/Traits/Selectable.cs

```diff
@@ -11,11 +11,17 @@
 
 namespace OpenRA.Traits
 {
-	[Desc("This actor is selectable. Defines bounds of selectable area, selection class and selection priority.")]
+	[Desc("This actor is selectable. Defines bounds of selectable area, selection class and selection priority. Defines whether actor is included in selection modes.")]
 	public class SelectableInfo : InteractableInfo
 	{
 		public readonly int Priority = 10;
 
+		[Desc("Include actor in select all mode.")]
+		public readonly bool IncludeInSelectAllMode = true;
+
+		[Desc("Include actor in low priority selection mode.")]
+		public readonly bool IncludeInSelectLowPriorityMode = false;
```

Comment:
Perhaps a personal style nit, but wouldn't it be better to expose these as flags?

```csharp
[Flags]
public enum SelectionPriorityModifiers
{
	All = 1,
	LowPriority = 2
}

[Desc("Allow selection priority to be modified using a hotkey.",
	"Valid values are All (raise priority when Ctrl pressed) and",
	"LowPriority (lower priority when Alt pressed).")]
public readonly SelectionPriorityModifiers PriorityModifiers = SelectionPriorityModifiers.All;
```

Reply:
I've changed the implementation to use flags.